plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/SearchResultXO.groovy

```diff
@@ -26,4 +26,26 @@ class SearchResultXO
   String groupId
   String artifactId
   String format
+
```

Comment:
should be able to just use the @EqualsAndHashcode anno http://groovy.codehaus.org/gapi/groovy/transform/EqualsAndHashCode.html